ngff image schema link moved

`pytest` fails at retreaving schemas from ngff github repo.

Example error:
`FAILED tests/tasks/test_unit_1536_well_conversion.py::test_1536_ome_zarr_conversion - requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://raw.githubusercontent.com/ome/ngff/main/0.4/schemas/image.schema`

This is likely due to https://github.com/ome/ngff/pull/308

@tcompa @jluethi I think the easiest option would be to change the schema links to: `https://github.com/ome/ngff/blob/7ac3430c74a66e5bcf53e41c429143172d68c0a4/schemas/image.schema`

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,6 @@
 **Note**: Numbers like (\#123) point to closed Pull Requests on the fractal-tasks-core repository.
 
-# 1.4.4
+# Unreleased
 
 * Testing:
     * Fix links to ome/ngff specs (\#930).

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,7 +3,7 @@
 # 1.4.4
 
 * Testing:
-    * Fix links to ome/ngff specs
+    * Fix links to ome/ngff specs (\#930).
 
 # 1.4.3
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,10 @@
 **Note**: Numbers like (\#123) point to closed Pull Requests on the fractal-tasks-core repository.
 
+# 1.4.4
+
+* Testing:
+    * Fix links to ome/ngff specs
+
 # 1.4.3
 
 * Tasks:

--- a/tests/tasks/_validation.py
+++ b/tests/tasks/_validation.py
@@ -19,11 +19,13 @@ from fractal_tasks_core.roi import (
 
 
 def validate_schema(*, path: str, type: str):
-
+    if __OME_NGFF_VERSION__ != "0.4":
+        raise ValueError(
+            "This test is only valid for OME-NGFF version 0.4, please update the test."
+        )
     file_path = pooch.retrieve(
         url=(
-            "https://raw.githubusercontent.com/ome/ngff/main/"
-            f"{__OME_NGFF_VERSION__}/schemas/{type}.schema"
+            f"https://raw.githubusercontent.com/ome/ngff/7ac3430c74a66e5bcf53e41c429143172d68c0a4/schemas/{type}.schema"
         ),
         known_hash=None,
     )

--- a/tests/tasks/_validation.py
+++ b/tests/tasks/_validation.py
@@ -21,11 +21,13 @@ from fractal_tasks_core.roi import (
 def validate_schema(*, path: str, type: str):
     if __OME_NGFF_VERSION__ != "0.4":
         raise ValueError(
-            "This test is only valid for OME-NGFF version 0.4, please update the test."
+            "This test is only valid for OME-NGFF version 0.4, "
+            "please update the test."
         )
     file_path = pooch.retrieve(
         url=(
-            f"https://raw.githubusercontent.com/ome/ngff/7ac3430c74a66e5bcf53e41c429143172d68c0a4/schemas/{type}.schema"
+            "https://raw.githubusercontent.com/ome/ngff/"
+            f"7ac3430c74a66e5bcf53e41c429143172d68c0a4/schemas/{type}.schema"
         ),
         known_hash=None,
     )

--- a/tests/test_unit_channels.py
+++ b/tests/test_unit_channels.py
@@ -126,10 +126,14 @@ def test_get_channel_from_list(testdata_path: Path):
 
 @pytest.fixture(scope="session")
 def omero_channel_schema():
+    
+    if __OME_NGFF_VERSION__ != "0.4":
+        raise ValueError(
+            "This test is only valid for OME-NGFF version 0.4, please update the test."
+        )
     file_path = pooch.retrieve(
         url=(
-            "https://raw.githubusercontent.com/ome/ngff/main/"
-            f"{__OME_NGFF_VERSION__}/schemas/image.schema"
+            "https://raw.githubusercontent.com/ome/ngff/7ac3430c74a66e5bcf53e41c429143172d68c0a4/schemas/image.schema"
         ),
         known_hash=None,
     )

--- a/tests/test_unit_channels.py
+++ b/tests/test_unit_channels.py
@@ -126,14 +126,16 @@ def test_get_channel_from_list(testdata_path: Path):
 
 @pytest.fixture(scope="session")
 def omero_channel_schema():
-    
+
     if __OME_NGFF_VERSION__ != "0.4":
         raise ValueError(
-            "This test is only valid for OME-NGFF version 0.4, please update the test."
+            "This test is only valid for OME-NGFF version 0.4, "
+            "please update the test."
         )
     file_path = pooch.retrieve(
         url=(
-            "https://raw.githubusercontent.com/ome/ngff/7ac3430c74a66e5bcf53e41c429143172d68c0a4/schemas/image.schema"
+            "https://raw.githubusercontent.com/ome/ngff/"
+            "7ac3430c74a66e5bcf53e41c429143172d68c0a4/schemas/image.schema"
         ),
         known_hash=None,
     )

